source/vstd/seq_lib.rs

```diff
@@ -1320,6 +1379,23 @@ proof fn to_multiset_build<A>(s: Seq<A>, a: A)
     }
 }
 
+proof fn to_multiset_remove<A>(s: Seq<A>, i: int)
```

Comment:
The Verus naming convention suggests prepending `lemma_` to the name of this function.

Reply:
Hmm, for some reason the existing `to_multiset_X` proof functions don't start with `lemma_`. So I see why you didn't do that here.